src/utils/postShowcase.ts

```diff
@@ -8,6 +8,15 @@ export default async (discordData: ShowcaseDiscordData, internalData: ShowcaseDa
   const { guild, channel, user, reaction } = discordData
   const { success, reason, project } = result
 
+  if (!process.env.PROJECT_LOG_CHANNEL) {
+    throw new Error('project log channel not found in env')
+  }
+  const logChannelId = process.env.PROJECT_LOG_CHANNEL
+  const logChannel = channel.client.channels.cache.get(logChannelId)
+  if (!logChannel || logChannel.type !== 'text') {
+    throw new Error(`could not find text log channel (channelID: ${logChannelId})`)
+  }
```

Comment:
Spacing, remove extraneous explicit env reference, mention in the thrown error if channel was not text
```suggestion
  const logChannelId = process.env.PROJECT_LOG_CHANNEL
  
  if (!logChannelId) {
    throw new Error('Project log channel not set')
  }
 
  const logChannel = channel.client.channels.cache.get(logChannelId)
  
  if (!logChannel || logChannel.type !== 'text') {
    throw new Error(`Could not fetch log channel (channelID=${logChannelId}, is text?=${logChannel && logChannel.type === 'text'})`)
  }
```